packages/gitmoji-changelog-core/src/index.js

```diff
@@ -31,10 +32,17 @@ function makeGroups(commits) {
 }
 
 function sanitizeVersion(version) {
-  return semver.valid(version, {
-    loose: false,
-    includePrerelease: true,
-  })
+  try {
+    return semver.valid(version, {
+      loose: false,
+      includePrerelease: true,
+    })
+  } catch (e) {
+    if (e.name === 'TypeError') {
+      throw FunctionalError(e.message)
```

Comment:
It's hard to know what kind of functional error it is. I think you should write a specific error message to  explain the error. It will help to understand the code and handled errors.